.github/workflows/frontend-ci.yml

```diff
@@ -54,6 +54,9 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - uses: ./.github/actions/pnpm-setup
+      - run: pnpm --filter @liam-hq/db supabase:start
```

Comment:
💭 
I was wondering about the necessity of this (supabase db start) yesterday.  
It seemed like this would launch many Docker containers, and I wanted to minimize it.  
However, it appears to complete in about 3 minutes, so I think we can proceed with this!  
If speed becomes an issue, we can reconsider it separately.

Reply:
You're right! I was wondering that too. For now, let me just keep on going!